components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```
@@ -249,9 +360,13 @@ def main(argv):
     comp_jobs_dir = clp_config.logs_directory / "comp-jobs"
     comp_jobs_dir.mkdir(parents=True, exist_ok=True)
 
-    logs_to_compress = _get_logs_to_compress(pathlib.Path(parsed_args.logs_list).resolve())
+    try:
+        logs_to_compress = _get_logs_to_compress(pathlib.Path(parsed_args.logs_list).resolve())
+        clp_input_config = _generate_clp_io_config(clp_config, logs_to_compress, parsed_args)
+    except Exception:
+        logger.exception(f"Failed to process input.")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)